src/main.rs

```diff
@@ -83,7 +91,12 @@ fn main() -> anyhow::Result<()> {
         unsupported => panic!("unsupported language: {}", unsupported),
     };
 
-    ir.synthesize(synthesizer.as_ref(), &mut output)?;
+    let stack_name = matches
+        .get_one::<String>("stack-name")
+        .map(String::as_str)
+        .unwrap_or("NoctStack");
```

Comment:
totally chill we're doing this like this for now. But at some point we need to incorporate cfn's get-template somewhere around this bit to get the stack name of an already deployed stack

Reply:
How does that mesh with the input and output being filepaths?

Reply:
nvm you did it above I just didnt notice. We're all gucci